### ISSUE TITLE
Can i req a few feature?

### ISSUE THREAD

**Issue by iFanpSGTS** [marktennyson/flask-express#141]
Please make .destroy()
Please remake the req.url (just print the endpoint like "/endpoint" don't print the full url)

**Comment by marktennyson** [comment]
Hi @iFanpSGTS, thanks for making features request.
Could you please elaborate the `destroy` method ?

**Comment by iFanpSGTS** [comment]
@marktennyson , the same like ExpressJS destroy method, u know it right.
"res.destroy()"

**Comment by marktennyson** [comment]
@iFanpSGTS 
Yeah, I got it. Shortly I will make a new release with this feature included. Thanks.

**Comment by iFanpSGTS** [comment]
@marktennyson 
Thank you very much for responding to my request, i request "destroy" method because, in flask i must return 204 code to send a blank response, and that so annoying, like i cant even go to the website just because that code, like its not send a blank response.
